src/service/viewport-impl.js

```diff
@@ -144,6 +144,14 @@ export class Viewport {
     this.binding_.onResize(this.resize_.bind(this));
 
     this.onScroll(this.sendScrollMessage_.bind(this));
+
+    // TODO(dvoytenko, #4894): Cleanup the experiment by moving this to CSS:
+    // `html {touch-action: pan-y}` (will require adding `amp-embedded` class).
+    // The enables passive touch handlers, e.g. for document swipe, since they
+    // no will longer need to try to cancel vertical scrolls during swipes.
+    if (viewer.isEmbedded() && isExperimentOn(this.win_, 'pan-y')) {
```

Comment:
Cool. Added more docs. Also, really, this is only needed when doc is embedded - otherwise there's no document swipe or things like that.